.github/workflows/release.yml

```diff
@@ -0,0 +1,65 @@
+  
+name: Release
+
+on:
+  push:
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+*"
+
+jobs:
+  analyze:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "12.x"
+      - uses: subosito/flutter-action@v1
+        with:
+          channel: "stable"
+      - name: Run CI
+        run: bash ./scripts/run_ci_tasks.sh -z
+  android:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "12.x"
+      - uses: subosito/flutter-action@v1
+        with:
+          channel: "stable"
+      - name: Run CI
+        run: bash ./scripts/run_ci_tasks.sh -a
+  ios:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
```

Comment:
is this copied from the android section? what do we need to set up java for here?

Reply:
This is just a direct port of the CI jobs - but yeah, this looks weird. I don't think analyze or ios need java set up, removed.

Reply:
Ok, let's leave it for now then. We can revisit it later.